libraries/cmd/movable.cpp

```diff
@@ -784,16 +784,16 @@ void Movable::Thrust(const Vector &amt1, bool afterburn) {
     }
 }
 
-// If in Travel mode (non-combat), speed is limited to x1000
+// If in Travel mode (non-combat), speed limit is multiplied by a power of 10.
```

Comment:
There's a lot to unpack here.
1. The engine refers to combat_mode_multiplier but shouldn't it be non_combat_mode_multiplier everywhere?
2. I assume I'll find a separate PR where you explain the change. In any case, this isn't correct. At best, the comment should say it defaults to x10.

Reply:
I'm trying to keep track of both the old way things were, and the new way things are now, until we figure out how they're supposed to be. That's why I added back the old code, commented out. It's also why I said more generically that the speed limit is "multiplied by a power of 10." It could be 100, or it could be 1000. For now, that is left unspecified.

The old code said combat_mode_multiplier. The new code says non_combat_mode_multiplier. It's pretty confusing, and I don't know which is correct.

Obviously, this situation isn't ideal. We need to figure out which it's supposed to be, on all of these counts, and change the code and comments accordingly. I just don't know enough to do that yet.

Reply:
Sorry. I misunderstood your comment. But is there something stopping us from using x25?!

Reply:
Not particularly. We could use x25 if we wanted lol